### PR TITLE
fix: small UX fix for lnode UI

### PR DESCRIPTION
**Pull request by l3gacyb3ta**
This will look prettier uwu :)

### CODE REVIEW

File: lnode-src/duckcoin.go [hunk 1]
```diff
@@ -382,7 +382,7 @@ func reCalcDifficulty() {
 	}
 	avg /= i
 	avgDur := time.Duration(avg)
-	fmt.Println("The average duration between blocks for the past 100 blocks was:", avgDur)
+	fmt.Println(fmt.Sprintf("The average duration between blocks for the past %d blocks was: %f", ReCalcInterval, avgDur))
```
Comment (quackduck) [inline lnode-src/duckcoin.go]: Has this been tested?

Comment (l3gacyb3ta) [reply lnode-src/duckcoin.go]: oops

Comment (quackduck) [reply lnode-src/duckcoin.go]: (I'm just not certain about the %f flag)

Comment (l3gacyb3ta) [reply lnode-src/duckcoin.go]: yeah it's wrong!

File: lnode-src/duckcoin.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ var (
 	NewestBlock util.Block
 	Balances    = make(map[string]int64)
 
-	ReCalcInterval   = 100
+	ReCalcInterval   = 1
```
Comment (quackduck) [inline lnode-src/duckcoin.go]:
```suggestion
	ReCalcInterval   = 100
```

**Pull request merged**